tests/rational.rs

```diff
@@ -766,76 +735,42 @@ pub mod test {
             &ruler1,
             "rational_replicate",
             "oopsla",
-            Limits {
-                iter: 2,
-                node: 150000,
-            },
             duration,
         );
         logger::write_output(
             &replicate_rules,
             &herbie,
             "rational_replicate",
             "herbie",
-            Limits {
-                iter: 2,
-                node: 150000,
-            },
             duration,
         );
 
         let start = Instant::now();
         let best_rules = best_enumo_recipe();
         let duration = start.elapsed();
 
-        logger::write_output(
-            &best_rules,
-            &ruler1,
-            "rational_best",
-            "oopsla",
-            Limits {
-                iter: 2,
-                node: 150000,
-            },
-            duration,
-        );
-        logger::write_output(
-            &best_rules,
-            &herbie,
-            "rational_best",
-            "herbie",
-            Limits {
-                iter: 2,
-                node: 150000,
-            },
-            duration,
-        );
+        logger::write_output(&best_rules, &ruler1, "rational_best", "oopsla", duration);
```

Comment:
Are we no longer interested in the replicate rules?

Reply:
no we still want all four- they're all here?